app/src/main/java/com/example/polina/socialnetwork/ServerAPI.java

```diff
@@ -61,6 +70,28 @@ public JSONObject saveProfile(String name, String birthday, String sex, String i
         }
     }
 
+
+
+    private JSONObject deleteRequest(Context context, String path) {
```

Comment:
Context shouldn't be used here whatsoever. Please remove it from each method. and do
CookieSyncManager.createInstance(context)  in the SNApp.onCreate.